src/probability_theory/probability_space.lean

```diff
@@ -0,0 +1,762 @@
+/-
+Copyright (c) 2021. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Martin Zinkevich (modified for mathlib by Hunter Monroe)
+ -/
+
+import measure_theory.measurable_space
+import measure_theory.measure_space
+
+/-!
+This file defines the basic concepts in probability theory.
+probability_space: a measure_space where the measure has a value of 1.
+event: a set that is measurable (defined based upon the measurable space),
+Pr[E]: the probability of an event.
+-/
+
+open measure_theory measurable_space
+open_locale big_operators classical
+
+namespace probability_theory
+
+/- An event is a measurable set in a measurable space that has a probability measure. Note:
+the definition below does not require that the measurable_space have volume one. -/
+@[derive [has_union,has_emptyc,has_compl, has_sdiff,has_union,has_inter,boolean_algebra]]
+def event (Ω : Type*) [measurable_space Ω] : Type* := {x : set Ω // measurable_set x}
+
+/- A probability space is a measure space with volume one. -/
+class probability_space (α : Type*) extends measure_space α := (volume_univ : volume (set.univ) = 1)
+
+export probability_space (volume_univ)
+
+@[simp]
+lemma probability_space.univ_one' {α:Type*} (Pα:probability_space α):
+  (@measure_theory.measure_space.volume α Pα.to_measure_space) (@set.univ α) = 1 :=
+begin
+  rw ← measure_theory.coe_to_outer_measure,
+  rw ← measure_theory.outer_measure.measure_of_eq_coe,
+  simp,
+  rw probability_space.volume_univ,
+end
```

Comment:
This statement is no different to `probability_space.volume_univ` that you already have, you just added all the `@`s that lean adds for you.

Reply:
Can consideration be given to my last commits, which delete probability_space.lean (which needs to be rewritten from scratch rather than using formal-ml), adds two probability_measure lemmas to measure_space.lean, and tidies up items out of order in that file? I am referring to commit bc5cdc3.